centaurCwlRunner/src/main/scala/centaur/cwl/Outputs.scala

```diff
@@ -85,7 +85,7 @@ object CwlOutputsFold extends Poly1 {
   }
 
   implicit def et: Case.Aux[cwl.ExpressionTool, Map[String, MyriadOutputType]] = at[cwl.ExpressionTool] {
-    _.outputs.map(output => output.id -> output.`type`).toMap
+    _.outputs.map(output => output.id -> output.`type`.get).toMap
```

Comment:
Weird... how did this ever compile?

Reply:
The type for the `ExpressionTool` outputs was not optional (the spec says it should http://www.commonwl.org/v1.0/Workflow.html#ExpressionToolOutputParameter)